### DISCUSSION TITLE
Cloud-hosted rate limit stopped working

### DISCUSSION THREAD

**Discussion by palm-ER** [FlowiseAI/Flowise#2262] [Q&A]
Is anyone else having issues with this?

When I go to `{{hosted_url}}/api/v1/ip` I get this:

`{
    "statusCode": 412,
    "success": false,
    "message": "Error: ipController.configureProxyNrInHostEnv - ip not provided!",
    "stack": {}
}`

In the chabot, instead of the rate limit message I get "Oops! There seems to be an error. Please try again."

**Comment by amansoniamazatic** [comment]
@palm-ER I also got the same error for latest version but its working fine for version 1.6.2 and below

**Comment by amansoniamazatic** [comment]
@HenryHengZJ please fix this issue
